changelog/fix_rails_advice.md

```diff
@@ -0,0 +1 @@
+* [#1263](https://github.com/rubocop/rubocop-rails/pull/1263): Fixes Rails configuration advice to protect against pretend generates ([@jherdman][])
```

Comment:
This changelog entry is not required. Can you remove it?

Reply:
Done!